### ISSUE TITLE
hb-graphite2 needs to be updated to use its new API

### ISSUE THREAD

**Issue by ebraminio** [harfbuzz/harfbuzz#1213]
```
hb-graphite2.cc:116:18: error: 'gr_make_face' is deprecated [-Werror,-Wdeprecated-declarations]
  data->grface = gr_make_face (data, &hb_graphite2_get_table, gr_face_preloadAll);
                 ^
/usr/include/graphite2/Font.h:143:1: note: 'gr_make_face' has been explicitly marked deprecated here
GR2_DEPRECATED_API gr_face* gr_make_face(const void* appFaceHandle/*non-NULL*/, gr_get_table_fn getTable, unsigned int faceOptions);
^
/usr/include/graphite2/Types.h:78:43: note: expanded from macro 'GR2_DEPRECATED_API'
  #define GR2_DEPRECATED_API  _gr2_tag_fn(_gr2_deprecated_flag _gr2_and _gr2_import_flag)
                                          ^
/usr/include/graphite2/Types.h:49:33: note: expanded from macro '_gr2_deprecated_flag'
  #define _gr2_deprecated_flag  deprecated
                                ^
1 error generated.
```

There is an alternative API and HarfBuzz should use that apparently.

Please remove `-Wno-deprecated-declarations` from `config.yml` after the fix also.

**Comment by behdad** [comment]
@mhosken

**Comment by ebraminio** [comment]
Fixed by #1215 and #1220
